Author: Andreas Hoenen <andreas@hoenen-terstappen.de>
Description: Improve CJK (Chinese/Japanese/Korean) support:
Let XSL configuration parameter cjk.font default to gkai (instead of
the cyberbit font not included in Debian).
--- a/xsl/lang.xsl
+++ b/xsl/lang.xsl
@@ -13,7 +13,7 @@
 <xsl:param name="latex.encoding">latin1</xsl:param>
 <xsl:param name="latex.engine.options"/>
 <xsl:param name="korean.package">CJK</xsl:param>
-<xsl:param name="cjk.font">cyberbit</xsl:param>
+<xsl:param name="cjk.font">gkai</xsl:param>
 <xsl:param name="xetex.font">
   <xsl:value-of select="concat('\setmainfont{',$body.font.family,'}&#10;')"/>
   <xsl:value-of select="concat('\setsansfont{',$sans.font.family,'}&#10;')"/>
